src/Microsoft.AspNetCore.SignalR/HubConnectionList.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public class HubConnectionList : IReadOnlyCollection<HubConnectionContext>
```

Comment:
Isn't the HubConnectionContext the real deal here though? This is basically a copy of ConnectionList and we need it because HubConnectionContext wraps ConnectionContext...